api.bs

```diff
@@ -855,6 +864,52 @@ if the user has opted out of collection of diagnostic data.
 * User ability to view the impression store and past report submissions.
 
 
+# HTTP API # {#http-api}
+
+\`<dfn http-header><code>Save-Impression</code></dfn>\` is a
+[=structured header/dictionary|Dictionary Structured Header=]
+set on a response requesting that the user agent invoke the
+<a method for=PrivateAttribution>saveImpression()</a> API.
+
+<pre class=example id=ex-save-impression-header>
+Save-Impression: conversion-site=advertiser.example, histogram-index=2, \
+  filter-data=12, lifetime-days=7
+</pre>
+
+The following keys are defined, corresponding to the members of
+the {{PrivateAttributionImpressionOptions}} dictionary passed to
+<a method for=PrivateAttribution>saveImpression()</a>.
+
+<dl dfn-for=save-impression>
+  <dt><code>conversion-site</code></dt>
+  <dd>
+    Value of <a dict-member for=PrivateAttributionImpressionOptions>conversionSite</a>,
+    a [=structured header/string=]. This key is required,
+    and must be a valid domain name.
```

Comment:
I'm not sure that `domain name` is precise enough here, based on https://w3c.github.io/ppa/#conversion-site. We might want to say `site` instead.